src/commands/Management/createMute.ts

```diff
@@ -15,17 +18,20 @@ export default class extends SkyraCommand {
 			description: language => language.tget('COMMAND_CREATEMUTE_DESCRIPTION'),
 			extendedHelp: language => language.tget('COMMAND_CREATEMUTE_EXTENDED'),
 			permissionLevel: PermissionLevels.Administrator,
-			requiredGuildPermissions: ['MANAGE_CHANNELS', 'MANAGE_ROLES'],
+			requiredGuildPermissions: ['MANAGE_CHANNELS', 'MANAGE_ROLES', 'USE_EXTERNAL_EMOJIS', 'ADD_REACTIONS', 'READ_MESSAGE_HISTORY'],
```

Comment:
I am against this. This will make the command unusable if Skyra doesn't have these permissions guild wise, which it doesn't even NEED for updating a configuration key. Imo, remove this and the react thing. Consistency please

Reply:
I've removed the extra permissions, but am keeping the react (with a fallback to text).

Reply:
Thatțs *fine*, but it still creates a massive inconsistency between command outputs :/